backend/src/main/java/gov/cdc/usds/simplereport/db/model/PatientPreferences.java

```diff
@@ -0,0 +1,52 @@
+package gov.cdc.usds.simplereport.db.model;
+
+import gov.cdc.usds.simplereport.db.model.auxiliary.TestResultDeliveryPreference;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.OneToOne;
+import org.hibernate.annotations.Type;
+
+@Entity
+public class PatientPreferences extends AuditedEntity {
+  @OneToOne private Person person;
```

Comment:
Potentially worth trying jimmy this around to use `@MapsId @JoinColumn(name="internal_id")`? Not sure what we would have to do to inheritance to make that work, though.

Reply:
Done :)

Didn't have to mess with inheritance at all